apps/demo-app/src/app/examples/search-filter/search-filter/search-filter-example.component.ts

```diff
@@ -29,7 +32,30 @@ export class SearchFilterExampleComponent {
   }
 
   searchFilterChange(searchParams: SearchFilterParams) {
-    this.fetchUsers(searchParams);
+    const errors: { [key: string]: string[] } = {};
```

Comment:
This example is a bit misleading. We want to show how to display server side validation errors in search filters. It should look like something like this:

```ts
import { setServerSideValidationErrors } from '@enigmatry/entry-components/validation'
....
....
this.fetchUsers(searchParams)
      .subscribe({
        error: (error: IValidationProblemDetails) => {
          /** Applies received server side validation errors to the search filters form */
          setServerSideValidationErrors(error, this.entrySearchFilterComponent.searchFilterForm);
        }
      });
```
I would move mocked server side validation response in service and return it from there.

Reply:
I will move it to the service